core/src/test/java/org/apache/accumulo/core/data/ValueTest.java

```diff
@@ -194,7 +194,7 @@ public void testEquals() {
   @Test
   public void testString() {
     Value v1 = new Value("abc");
-    Value v2 = new Value("abc".getBytes(UTF_8));
+    Value v2 = new Value("abc");
```

Comment:
This one shouldn't be changed, because it's explicitly testing the `CharSequence` and `byte[]` equivalence.